TYPO3.Neos/Configuration/NodeTypes.yaml

```diff
@@ -2,6 +2,8 @@
 'TYPO3.Neos:Node':
   label: "${String.cropAtWord(String.trim(String.stripTags(String.pregReplace(q(node).property('title') || q(node).property('text') || ((I18n.translate(node.nodeType.label) || node.nodeType.name) + (node.autoCreated ? ' (' + node.name + ')' : '')), '/<br\\W*?\\/?>|\\x{00a0}|[[^:print:]]|\\s+/u', ' '))), 100, '...')}"
   abstract: TRUE
+  options:
+    fusionPrototypeGenerator: TYPO3\Neos\Domain\Service\DefaultPrototypeGenerator
```

Comment:
I'm debating if the key `options` is a good idea here.. options is pretty broad and all of this configuration is somehow options.. would a key like `fusion` and then prototypeGenerator on next level make more sense?